### PR TITLE
Mica Sample Update

### LINKED ISSUE #1286: Update Mica XAML and C# samples
### Issue type

enhancement

### Which version of the app?

WinUI 3 Gallery

### Description

Now that Mica is in W.A.SDK 1.3, we should mention the xaml on the Backdrops page. And/or link to https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.micabackdrop

### Screenshots

image

### PR DESCRIPTION
**Pull request by karkarl**
## Description

Updating the Mica sample now that it is available in WASDK 1.3, and way easier to implement!

_Note: This is a port from internal repo PR - 8915253_

## Motivation and Context

Fixes #1286

## Screenshots (if appropriate):

image

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/SamplePages/SampleBuiltInSystemBackdropsWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    x:Class="AppUIBasics.SamplePages.SampleBuiltInSystemBackdropsWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <StackPanel Margin="20" Spacing="10">
+            <StackPanel Orientation="Horizontal">
```
Comment (niels9001) [inline WinUIGallery/SamplePages/SampleBuiltInSystemBackdropsWindow.xaml:11]:
```suggestion
            <StackPanel Orientation="Horizontal" Spacing="4">
```

File: WinUIGallery/SamplePages/SampleBuiltInSystemBackdropsWindow.xaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                <TextBlock Text="Current backdrop: " />
+                <TextBlock x:Name="tbCurrentBackdrop" />
```
Comment (niels9001) [inline WinUIGallery/SamplePages/SampleBuiltInSystemBackdropsWindow.xaml:13]:
```suggestion
                <TextBlock x:Name="tbCurrentBackdrop" FontWeight="SemiBold" />
```

File: WinUIGallery/ControlPages/SystemBackdropsPage.xaml [hunk 3]
```diff
@@ -19,14 +19,19 @@
       mc:Ignorable="d">
 
     <StackPanel>
+        <local:ControlExample HeaderText="Create a window with a built-in Mica system backdrop."
```
Comment (niels9001) [inline WinUIGallery/ControlPages/SystemBackdropsPage.xaml:22]: @karkarl I'd include a XAML snippet here as well now that's fully supported? It's probably how most devs would (like) to use it?

**Review by niels9001 — APPROVED** [review]
LGTM! See minor comment - having a XAML snippet would be a nice addition!

I pushed a commit prettyfing the sample pages a bit :)

File: WinUIGallery/ControlPages/SystemBackdropsPage.xaml (hunk 3 above)
Comment (karkarl) [reply WinUIGallery/ControlPages/SystemBackdropsPage.xaml:22]: What should be in the Xaml snippet? I can't think of anything relevant to mica implementation to put in there.

Comment (niels9001) [reply WinUIGallery/ControlPages/SystemBackdropsPage.xaml:22]:
```XAML
<Window.SystemBackdrop>
     <MicaBackdrop/>
</Window.SystemBackdrop>
```

and

```XAML
<Window.SystemBackdrop>
    <DesktopAcrylicBackdrop/>
</Window.SystemBackdrop>
```

Comment (karkarl) [reply WinUIGallery/ControlPages/SystemBackdropsPage.xaml:22]: Added!

**Comment by mdtauk** [comment]
Did you also include the sample with one for Mica Alt?
```xaml
<Window.SystemBackdrop>
    <MicaBackdrop Kind="BaseAlt"/>
</Window.SystemBackdrop>
```

**Pull request merged**